android-components/components/browser/menu/src/test/java/mozilla/components/browser/menu/BrowserMenuHighlightTest.kt

```diff
@@ -36,12 +36,13 @@ class BrowserMenuHighlightTest {
     @Suppress("Deprecation")
     @Test
     fun `classic highlight effect converts background tint`() {
+        val colorId = mozilla.components.ui.colors.R.color.photonRed50
```

Comment:
I think we would prefer to import path and use an appropriate alias instead of inlining the full path.

Reply:
Dealer's choice, but I think the utility of an alias makes more sense the more times it's used in a file. Using an alias just for one or two instances with no line length issue could be argued in the name of consistency, though.

Reply:
I think I feel strongly enough about this that I would suggest adding the imports. I would like to avoid inadvertently encouraging others to think it's okay to use the full namespace.

Reply:
Before proceeding with my suggested, I will bring this change up with the team just for general awareness. I don't foresee any concerns coming out of it, but I just want to avoid encouraging any additional changes before knowing for sure if we'll accept the proposed changes.

Reply:
I'd agree, @gabrielluong - inlining the path seems unnecessarily messy. I'd personally prefer the style of having paths imported (and, optionally, include an alias for the import - particularly if there are name collisions that require that).

Reply:
Actually, I think I'd phrase my preference as even more strict: always import the path (up to but not including `.R`, so every resource can start with the standard `R.`- prefix), and allow optional aliases but _only_ when doing so adds some useful clarity to which resource you're referencing.

Am I correct in believing that Android Studio adds the import statements for you automatically with the default text completion? If so, that seems like it would be the option that requires the least amount of thought while developing.

Reply:
> Actually, I think I'd phrase my preference as even more strict: always import the path (up to but not including .R, so every resource can start with the standard R.- prefix)

I don't think that Kotlin allows this. When attempting to import e.g. `mozilla.components.ui.colors` instead of `mozilla.components.ui.colors.R` I get: `Packages cannot be imported`. According to the [docs](https://kotlinlang.org/docs/packages.html#imports) it seems like you can't import packages like you can in e.g. Python. One could use a wildcard, but that unnecessarily brings along all other references as well (which may result in more conflicts).

> Am I correct in believing that Android Studio adds the import statements for you automatically with the default text completion?

I had to enable the option, but the option does exist. I don't think text completion for `R` classes works without first getting the import correct. though.

Reply:
I agree with using alias for the consistency. Though we have files with only one or two changes now, and seems reasonable to skip using alias for them; I believe it is possible in the future that people will refer to the resources more and may populate the files with more `full/path` resource files.